pages/components/Register.tsx

```diff
@@ -0,0 +1,100 @@
+import axios, { AxiosError } from 'axios';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+
+const Register = () => {
+    const [name, setName] = useState('');
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [showModal, setShowModal] = useState(false); 
+    const router = useRouter();
+
+    const handleSubmit = async (e: React.FormEvent) => {
+        e.preventDefault();
+
+        try {
+            const response = await axios.post('/api/auth/Register', { name, email, password }, { headers: { 'Content-Type': 'application/json' } } );
+            localStorage.setItem('token', response.data);
+            router.push('/components/Login');
```

Comment:
let's use `/login` for this page route. `/components` should house UIs of a page not a whole page